app/assets/javascripts/payments.js

```diff
@@ -0,0 +1,3 @@
+function braintree_setup() {
```

Comment:
Identifier 'braintree_setup' is not in camel case.<br>'braintree_setup' is defined but never used.